### PR TITLE
Added possibility to pass constructor arguments with asObject

### PR DESCRIPTION
**Pull request by teddybeermaniac**
Added optional $arguments argument to asObject method which, if is an array, will be passed as ctorargs to setFetchMode.

### CODE REVIEW

File: FluentPDO/BaseQuery.php [hunk 1]
```diff
@@ -19,6 +19,9 @@
 	/** @var bool */
 	private $object = false;
 
+	/** @var array of object's constructor arguments */
+	private $object_arguments = null;
```
Comment (hoshsadiq) [inline FluentPDO/BaseQuery.php]: use camelCasing

File: tests/63-from-id-asobject-user-with-arguments.phpt [hunk 2]
```diff
@@ -0,0 +1,23 @@
+--TEST--
+from($table, $id) as User class with arguments
+--FILE--
+<?php
+include_once dirname(__FILE__) . "/connect.inc.php";
+/* @var $fpdo FluentPDO */
+class UserArgument { public $id, $country_id, $type, $name, $argument; public function __construct($argument) { $this->argument = $argument; } }
```
Comment (hoshsadiq) [inline tests/63-from-id-asobject-user-with-arguments.phpt]: it doesn't hurt to format this to help keep maintenance easy

File: tests/64-from-id-asobject-user-with-arguments2.phpt [hunk 3]
```diff
@@ -0,0 +1,22 @@
+--TEST--
+from($table, $id) as User class with arguments
+--FILE--
+<?php
+include_once dirname(__FILE__) . "/connect.inc.php";
+/* @var $fpdo FluentPDO */
+class UserArgument2 { public $id, $country_id, $type, $name; public function __construct($name) { $this->name = $name; } }
```
Comment (hoshsadiq) [inline tests/64-from-id-asobject-user-with-arguments2.phpt]: same here

**Comment by cbornhoft** [comment]
Closing this PR. This doesn't have much to do with queries or PDO, so if anyone requires this functionality, I recommend they extend Fluent to achieve this.

**Pull request closed without merging**